### DISCUSSION TITLE
How to resample PCD to fixed pitch

### DISCUSSION THREAD

**Discussion by alfa159sw** [isl-org/Open3D#4433] [General]
Hello! I would like to resample PCD data such as attachments at a predetermined pitch or space.
Any Idea please.
[test_outlier.zip](https://github.com/isl-org/Open3D/files/7723819/test_outlier.zip)

**Comment by ssheorey** [comment]
Hi @alfa159sw you can use `VoxelDownSample` to resample point clouds to a predetermined spacing. You will get a single point in each voxel and you can set the voxel size. Here's an example:

http://www.open3d.org/docs/latest/python_example/geometry/point_cloud/index.html#point-cloud-voxel-downsampling-py
